about.html

```diff
@@ -0,0 +1,191 @@
+                <a class="nav-link" href="#">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">News</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">CC Campaign</a>
+              </li>
+            </ul>
+            <form class="d-flex mt-3" role="search">
+              <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+              <button class="btn btn-outline-success" type="submit">Search</button>
+            </form>
+          </div>
+        </div>
+      </div>
+    </nav>
+
+  </div>
+
+  <nav class="nav-desktop2 ">
+
+    <div class="nav-desktop2-margin">
+      <a href="https://www.facebook.com"><i class="bi bi-facebook"></i></a>
+      <a href="https://twitter.com/?lang=es"><i class="bi bi-twitter"></i></a>
+      <a href="about.html">Logout</a>
+      <a href="#about">Mypage</a>
+      <a href="#contact">English</a>
+    </div>
+
+    
+
+
+
+  </nav>
+  <nav class="nav-desktop ">
+    <h5 class="offcanvas-title" id="offcanvasNavbarLabel"> <a href="index.html"><img src="./images/Logo.png" alt=""></a> </h5>
+    <a href="#portfolio">About </a>
+    <a href="#about">Program</a>
+    <a href="#contact">Join</a>
+    <a href="#portfolio">Sponsor</a>
+    <a href="#portfolio">News</a>
+    <a href="#portfolio">CC Campaign</a>
+
+
+  </nav>
+  <section class="text-center background-intro">
+    <div class="container-md cocogoose">
+    
+      <div class="margin-text1">
+        <h1 class="h1-intro">WEB PROGRAMMING COURSE'S FOR EVERYONE 2023</h1>
+      </div>
+      
+    </div>
+
+    <div class="margin-text2 about-margin-text2">
+      <div class="containe containerpro-1">
+        <p>
+          The Global Summit brings together the community of exeperts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two. This year, we hope to expand our invitation list including organizations and individuals.
+        </p>
+      </div>
+    </div>
+
+
+    
+    <div class="container-md">
+      
+      <p class="p-intro2"> Please contact us per Email for any further questions about Web Programming Courses 2023!</p>
+      <p class="p-intro2">  </p>
+      <p class="p-intro2"><strong>webprogramming@gmail.com</strong></p>
+    </div>
+  </section>
+
+<section>
+    <div class="container">
+        <p class="text-center p-intro-section"><strong>Creative Commons Global Summit 2023 Logo</strong> </p>
+        <p class="text-center p-intro-section"><strong></strong> </p>
+        <hr class="hr">
+        
+        <p class="text-center p-intro2"> The logo of CC Global Summit 2015 was decided</p>
+        <p class="text-center p-intro2">through the logo competition from 8. June to 7 July.</p>
+
+        <div>
+            <img class="logo-img"  src="./images/Logo2.png" alt="">
+        </div>
+
+    </div>
+
+</section>
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 adapt the content of this section to the topic of your page, it is web programming courses instead of creative Commons for a better UI.
- **[OPTIONAL]** Kindly correct the position of the text on the mobile version for a better UI. Even the logo is shown in the mobile version but not in the desktop version. I highly recommend creating your own logo according to the topic of your page. 👇🏽 

| DESKTOP VERSION | MOBILE VERSION |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/233487375-63d97ce4-d30e-40bd-b55d-0a45a6419ab3.png) | ![image](https://user-images.githubusercontent.com/58642949/233486999-77808a6c-1f53-4028-ba3e-4664b25703f7.png) |

Reply:
![image](https://user-images.githubusercontent.com/111823606/233501279-e718a50c-7847-43c9-808b-84b8d5712169.png)
Done, thank you for notice that!!!